src/reducers/computeRenderedRows.js

```diff
@@ -189,9 +192,10 @@ function calculateRenderedRowRange(state, scrollAnchor) {
  *   firstBufferIdx: number,
  *   firstViewportIdx: number,
  * }} rowRange
+ * @param {boolean=} computeWithinViewportOnly
  * @private
  */
-function computeRenderedRowOffsets(state, rowRange) {
+function computeRenderedRowOffsets(state, rowRange, computeWithinViewportOnly = false) {
```

Comment:
Do we ever call this without the 3rd parameter?  If not, let's not make it optional